test_backend.py

```diff
@@ -117,13 +123,201 @@ def test_robot_is_repaired(damages_before, tile, damages_after):
                          (RepairTile(None, None, [{'value': False}]), (1, 1)),
                         ])
 def test_robot_changed_start_coordinates(tile, coordinates_after):
+    """
+    When robot is on RepairTile with special property, he changes it starting coordinates to the tile coordinates.
```

Comment:
` ... , he changes it starting coordinates to the tile coordinates.` Tady máš ve větvě dva podměty.